src/modules/75-cf/pages/environment-details/EnvironmentDetailsBody.tsx

```diff
@@ -245,18 +247,23 @@ const EnvironmentSDKKeys: React.FC<{ environment: EnvironmentResponseDTO }> = ({
       )}
 
       {emptyData && (
-        <Layout.Vertical width="100%" height="100%" flex={{ align: 'center-center' }} spacing="medium">
-          <Text>{getEnvString('apiKeys.noKeysFound')}</Text>
-          <AddKeyDialog
-            primary
-            environment={environment}
-            onCreate={(newKey: ApiKey, hideCreate) => {
-              setRecents([...recents, newKey])
-              hideCreate()
-              refetch()
-            }}
-          />
-        </Layout.Vertical>
+        <Container flex style={{ flexDirection: 'column' }}>
+          <NoData
+            imageURL={EmptySDKs}
+            title={getString('cf.environments.apiKeys.noKeysFoundTitle')}
+            message={<String useRichText stringID="cf.environments.apiKeys.noKeysFoundMessage" />}
```

Comment:
Since `message` was already in use, if you go with `description` (or something else like that), then it would look something like this.
```suggestion
            message={getString('cf.environments.apiKeys.noKeysFoundTitle')}
            description={<String useRichText stringID="cf.environments.apiKeys.noKeysFoundMessage" />}
```